### PR TITLE
services/horizon: Move FeeStatsHandler to actions.

### PR DESCRIPTION
**Pull request by abuiles**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://www.stellar.org/developers/reference/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Move FeeStatsHandler to actions and use new approach to action handling.

### CODE REVIEW

File: services/horizon/internal/actions/fee_stats.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	"github.com/stellar/go/services/horizon/internal/operationfeestats"
+	"github.com/stellar/go/support/render/httpjson"
+	"github.com/stellar/go/support/render/problem"
+)
+
+// FeeStatsHandler is the action handler for the /fee_stats endpoint
+type FeeStatsHandler struct {
+}
+
+func (handler FeeStatsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```
Comment (2opremio) [inline services/horizon/internal/actions/fee_stats.go:17]: Why not use a Raw Handle?

Comment (2opremio) [reply services/horizon/internal/actions/fee_stats.go:17]: Actually ... why can't you use `objectActionHandler`?

Comment (2opremio) [reply services/horizon/internal/actions/fee_stats.go:17]: (or a variant if needed)

Comment (abuiles) [reply services/horizon/internal/actions/fee_stats.go:17]: @2opremio you are right - we can use `objectActionHandler` --I  forgot about it :)

**Comment by abuiles** [comment]
@2opremio updated to use objectActionHandler :)

**Pull request merged**